### PR TITLE
[AS-2192] [AS-2191] Fix spotinst crash

### PR DESCRIPTION
**Pull request by hongshibao**

### CODE REVIEW

**Review by swimablefish — COMMENTED** [review]
Please also merge to release-1.13-eks

**Review by swimablefish — APPROVED** [review]
LGTM

File: cluster-autoscaler/cloudprovider/spotinst/spotinst_manager.go [hunk 1]
```diff
@@ -346,6 +348,40 @@ type groupTemplate struct {
 	Tags         []*aws.Tag
 }
 
+func (mgr *CloudManager) inferInstanceType(instanceTypeName string) *instanceType {
```
Comment (houdejun214) [inline cluster-autoscaler/cloudprovider/spotinst/spotinst_manager.go:351]: 👍

**Review by houdejun214 — APPROVED** [review]
LGTM

**Review by emman27 — APPROVED** [review]

**Pull request merged**